pom.xml

```diff
@@ -24,7 +24,7 @@
 
     <properties>
         <!-- Select the GraalVM version to use. -->
-        <graalvm.version>23.1.0</graalvm.version>
+        <graalvm.version>23.1.2</graalvm.version>
```

Comment:
I guess we need to wait until its released?

Reply:
Yes, we need to wait.  The uber JAR does not work with the graalvm-23.1.1 due to the classpath isolation.